### ISSUE TITLE
Bad usage for: atomic run --help 

### ISSUE THREAD

**Issue by jlebon** [projectatomic/atomic#327]
```
# atomic run --help
usage: atomic run [-h] [-n NAME] [--spc] [--display] image ...

positional arguments:
  image                 container image
  command               command to execute within the container. If container
                        is not running, command is appendedto the image run
                        method

optional arguments:
  -h, --help            show this help message and exit
  -n NAME, --name NAME  name of container
  --spc                 use super privileged container mode: '/usr/bin/docker
                        run -t -i --rm --privileged -v /:/host -v /run:/run -v
                        /etc/localtime:/etc/localtime --net=host --ipc=host
                        --pid=host -e HOST=/host -e NAME=${NAME} -e
                        IMAGE=${IMAGE} ${IMAGE}'
  --display             preview the command that /bin/atomic would execute

atomic run defaults to the following command, if image does not specify LABEL
run '/usr/bin/docker create -t -i --name ${NAME} ${IMAGE}'
```

The positional arguments list `command` but the usage string doesn't have the word `command` anywhere. To someone unfamiliar with `atomic`, it's not obvious that `command` refers to the `...` at the end (in fact it makes it seem like `atomic run` can start multiple images at once).

I'd suggest something like:

```
usage: atomic run [-h] [-n NAME] [--spc] [--display] image [command] [arg...]

positional arguments:
  image                 container image
  command               command to execute within the container. If container
                        is not running, command is appendedto the image run
                        method
  arg                   arguments to pass to command
```

I'm guessing this is automatically output by argparse, so it will probably require reorganizing the run parser.

**Comment by rhatdan** [comment]
Yes I would think this would require a code change, if possible.

**Comment by rhatdan** [comment]
Fixed in atomic-1.13
